aleph_message/models/__init__.py

```diff
@@ -80,7 +80,12 @@ class ItemHash(str):
     item_type: ItemType
 
     # When overriding str, override __new__ instead of __init__.
-    def __new__(cls, value, item_type: ItemType):
+    def __new__(cls, value):
```

Comment:
No type hint for value ?
```suggestion
    def __new__(cls, value: str):
```